### PR TITLE
next stage in porting CI from Travis to GitHub actions

### PR DESCRIPTION
**Pull request by dgrove-oss**
We were not running any actual tests on this repo, so can trivially drop travis.  However, we really should be running scan code, so fixed the handful of violations and add a GHA workflow to run scan code.

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    branches: [ master ]
+    types: [ opened, synchronize, reopened ]
+jobs:
+  scancode:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Scan Code
+        uses: apache/openwhisk-utilities/scancode@master
```
Comment (style95) [inline .github/workflows/ci.yaml:33]: 👍

File: README.md [hunk 2]
```diff
@@ -20,7 +20,6 @@
 # Apache OpenWhisk Project Release Management
 
 [![License](https://img.shields.io/badge/license-Apache--2.0-blue.svg)](http://www.apache.org/licenses/LICENSE-2.0)
-[![Build Status](https://travis-ci.com/apache/openwhisk-release.svg?branch=master)](https://travis-ci.com/github/apache/openwhisk-release)
```
Comment (style95) [inline README.md:23]: Could we add the status badge via Github workflow too or is this removed for some reason?
https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/adding-a-workflow-status-badge

**Review by style95 — APPROVED** [review]
LGTM with a minor query.

File: README.md (hunk 2 above)
Comment (dgrove-oss) [reply README.md:23]: Once the workflow is running, I plan to add the badge in a follow up PR.

**Pull request merged**